File: by_hand/lib/models/rate.dart
class Rates{
  int id;
  int productId;
  String userproduct;
  double valueRate;

  Rates();

  Rates.fromJson(Map<String,dynamic>json){
    this.id=json['id'];
    this.productId=json['product_id'];
    this.valueRate=json['value_rate'];
  }
}

File: by_hand_api/src/models/rate.js
import { Model } from 'sequelize';


export default (sequelize, DataTypes) => {
  class Rates extends Model {
    
    static associate(models) {
      // define association here
      Rates.belongsTo(models.Products,{foreignKey:"product_id"})

    }
  };
  Rates.init({
    product_id: DataTypes.INTEGER,
    value_rate: DataTypes.DOUBLE,
    user_product:DataTypes.STRING,
    deletedAt: 'destroyTime'

  }, {
    paranoid: true,
    sequelize,
    tableName:'rate',
    modelName: 'Rates',
  });
  return Rates;
};